exercises/practice/linked-list/.meta/proof.ci.wren

```diff
@@ -0,0 +1,41 @@
+class LinkedList {
+  construct new() {
+    _list = []
+  }
```

Comment:
Using a list object to hold the data makes this exercise trivial.

I'd prefer if we did the same as javascript: tell students _not_ to use a list to solve it: https://github.com/exercism/javascript/blob/main/exercises/practice/linked-list/.docs/instructions.append.md

Reply:
Fair enough

Reply:
Yes, the proofs  - **while not needing to be perfect or fully optimized** - should at least be a "reasonable good faith" attempt to solve the problem as specified - which in this case requires writing an actual linked list.  Also they should preferably use Wren-isms and consistent Wren-like naming when they can. 

@Atomic-Germ Thanks for keeping them coming though! :-)  Awesome to see the Wren track growing so fast lately.